examples/example-gateway/clients/contacts/client-config.yaml

```diff
@@ -9,5 +9,6 @@ config:
       - success
   idlFile: clients/contacts/contacts.thrift
   idlFileSha: '{{placeholder}}'
+  customInterface: Client
```

Comment:
Change the Interface name to CustomClient to test if this code here works or nor. otherwise, we won't be able to distinguish between default and custom case.

Reply:
I tested this locally and it works. As in, CustomClient for /contacts/ will not work since contacts is not a custom client, if it was, it would be generating the mock correctly.

Reply:
testing locally is never enough. It needs continuous testing through tests to know if there is any regression in future.